app/assets/v2/js/grants/funding.js

```diff
@@ -157,46 +231,49 @@ function sideCartRowForGrant(grant, index) {
   return cartRow;
 }
 
-
 function tokenOptionsForGrant(grant) {
-
   var network = document.web3network;
 
   if (!network) {
     network = 'mainnet';
   }
 
   // let tokenDataList = tokens(network);
-  let tokenDataList = allTokens.filter((token) => token.network === networkName || 'mainnet');
+  let tokenDataList = allTokens.filter(
+    (token) => token.network === networkName || 'mainnet'
+  );
   let tokenDefault = 'ETH';
 
   if (grant.tenants && grant.tenants.includes('ZCASH')) {
-    tokenDataList = tokenDataList.filter(token => token.chainId === 123123);
+    tokenDataList = tokenDataList.filter((token) => token.chainId === 123123);
     tokenDefault = 'ZEC';
-  } if (grant.tenants && grant.tenants.includes('CELO')) {
-    tokenDataList = tokenDataList.filter(token => token.chainId === 42220);
+  }
+  if (grant.tenants && grant.tenants.includes('CELO')) {
+    tokenDataList = tokenDataList.filter((token) => token.chainId === 42220);
     tokenDefault = 'CELO';
   } else if (grant.tenants && grant.tenants.includes('ZIL')) {
-    tokenDataList = tokenDataList.filter(token => token.chainId === 102);
+    tokenDataList = tokenDataList.filter((token) => token.chainId === 102);
     tokenDefault = 'ZIL';
   } else if (grant.tenants && grant.tenants.includes('HARMONY')) {
-    tokenDataList = tokenDataList.filter(token => token.chainId === 1000);
+    tokenDataList = tokenDataList.filter((token) => token.chainId === 1000);
     tokenDefault = 'ONE';
   } else if (grant.tenants && grant.tenants.includes('BINANCE')) {
-    tokenDataList = tokenDataList.filter(token => token.chainId === 56);
+    tokenDataList = tokenDataList.filter((token) => token.chainId === 56);
     tokenDefault = 'BNB';
   } else if (grant.tenants && grant.tenants.includes('POLKADOT')) {
-    tokenDataList = tokenDataList.filter(token => token.chainId === 58);
+    tokenDataList = tokenDataList.filter((token) => token.chainId === 58);
     tokenDefault = 'DOT';
   } else if (grant.tenants && grant.tenants.includes('KUSAMA')) {
-    tokenDataList = tokenDataList.filter(token => token.chainId === 59);
+    tokenDataList = tokenDataList.filter((token) => token.chainId === 59);
     tokenDefault = 'KSM';
   } else {
-    tokenDataList = tokenDataList.filter(token => token.chainId === 1);
+    tokenDataList = tokenDataList.filter((token) => token.chainId === 1);
```

Comment:
^ could we undo these @apbendi

Reply:
hey @thelostone-mc, I believe I made this change due to linting errors, but happy to undo if that's not an issue.